proposals/devcontainer-templates.md

```diff
@@ -47,22 +47,20 @@ The `options` property contains a map of option IDs and their related configurat
       "type": "string",
       "description": "Description of the option",
       "proposals": ["value1", "value2"],
-      "default": "value1",
-      "replaceIn": [".devcontainer.json"],
```

Comment:
Removed `replaceIn` as the tooling can find and replace in all the files within the Template folder.